sdk/src/main/scala/com/horizen/SidechainSettings.scala

```diff
@@ -25,10 +25,15 @@ case class WalletSettings(
                         genesisSecrets: Seq[String]
                          )
 
+case class MainchainSettings(
+                              path: String
```

Comment:
related to path usage in RpcMainchainApi. Name should be different

Reply:
Done.